src/java/org/apache/ivy/plugins/resolver/IBiblioResolver.java

```diff
@@ -191,7 +191,34 @@ public void endElement(String uri, String localName, String qName)
                         
                         return rev;
                     }
-                } else {
+                } else if (root.startsWith("file:")) {
+                    // sbt change, driven by https://github.com/sbt/sbt/issues/1616
+                    String metadataLocalLocation = IvyPatternHelper.substitute(
+                    root + "[organisation]/[module]/[revision]/maven-metadata-local.xml", mrid);
+                    Resource metadataLocal = getRepository().getResource(metadataLocalLocation);
+                    if (metadataLocal.exists()) {
+                        Message.verbose("maven-metadata-local found: " + metadataLocal);
+                        metadataStream = metadataLocal.openStream();
+                        final StringBuffer localCopy = new StringBuffer();
+                        XMLHelper.parse(metadataStream, null, new ContextualSAXHandler() {
+                            public void endElement(String uri, String localName, String qName)
+                                    throws SAXException {
+                                if ("metadata/versioning/snapshot/localCopy".equals(getContext())) {
+                                    localCopy.append(getText());
+                                }
+                                super.endElement(uri, localName, qName);
+                            }
+                        }, null);
+                        if (localCopy.length() > 0) {
```

Comment:
Given `localCopy` here is a StringBuffer, `length() > 0` means it has some content, which is redundant given the `localCopy.toString() == "true"` below.